docs/src/main/tut/core-concepts.md

```diff
@@ -191,22 +191,20 @@ We'll see more details about these and other annotations in the following sectio
 
 ## Compression
 
-[frees-rpc] allows you to compress the datas in our services. We can enable this compression in the server or in the client side.
+[frees-rpc] allows you to compress the data whose we are sending in our services. We can enable this compression either on the server or client side.
```

Comment:
Grammar fix:
`[frees-rpc] allows us to compress the data we are sending in our services. We can enable this compression either on the server or the client side.`